R/ml_recommendation_als.R

```diff
@@ -263,7 +280,16 @@ utils::globalVariables("explode")
 #'
 #' @export
 ml_recommend <- function(model, type = c("items", "users"), n = 1) {
-  if (spark_version(spark_connection(model)) < "2.2.0")
+
+  if(inherits(model, "ml_model_als")){
```

Comment:
Could we refactor this to check for the version and throw the error just once? e.g. `version <- if (inherits ml model) {} else {}` then check `version`.